mappings/net/minecraft/network/listener/ClientPlayPacketListener.mapping

```diff
@@ -197,3 +197,7 @@ CLASS net/minecraft/class_2602 net/minecraft/network/listener/ClientPlayPacketLi
 		ARG 1 packet
 	METHOD method_39025 onSimulationDistance (Lnet/minecraft/class_6682;)V
 		ARG 1 packet
+	METHOD method_43595 onSignedChatMessage (Lnet/minecraft/class_7438;)V
+		ARG 1 packet
+	METHOD method_43596 onChatMessage (Lnet/minecraft/class_7439;)V
```

Comment:
Should we name it `onUnsignedChatMessage` now that we have a signed version?

Reply:
Note that this is also used by things like death message or join/leave messages, which aren't usually referred as chat messages.

Reply:
Maybe we can still use the old `onGameMessage`